resources/views/components/incident.blade.php

```diff
@@ -3,10 +3,10 @@
     'incidents',
 ])
 
-<div class="relative flex flex-col gap-5">
-    <h3 class="text-xl font-semibold">{{ $date }}</h3>
+<div class="relative flex flex-col gap-5" x-data="{ forDate: new Date(@js($date)) }">
+    <h3 class="text-xl font-semibold" x-html="forDate.toLocaleDateString()"></h3>
```

Comment:
Any specific reason to use `x-html` instead of `x-text`?

Reply:
Mistake on my part. Would love a PR to fix!

Reply:
@MrPunyapal actually, if you do this, we should use the `time` element so the real timestamp is always rendered.